api.go

```diff
@@ -217,6 +220,44 @@ func (h *Headscale) RegistrationHandler(
 			return
 		}
 
+		// Check if the node is waiting for interactive login
+		//
+		// TODO(juan): We could use this field to improve our protocol implementation,
+		// and hold the request until the client closes it, or the interactive
+		// login is completed (i.e., the user registers the machine).
+		// This is not implemented yet, as it is no strictly required. The only side-effect
+		// is that the client will hammer headscale with requests until it gets a
+		// successful RegisterResponse.
+		if registerRequest.Followup != "" {
+			if _, ok := h.registrationCache.Get(NodePublicKeyStripPrefix(registerRequest.NodeKey)); ok {
+				log.Debug().
+					Caller().
+					Str("machine", registerRequest.Hostinfo.Hostname).
+					Str("NodeKey", registerRequest.NodeKey.ShortString()).
+					Str("OldNodeKey", registerRequest.OldNodeKey.ShortString()).
+					Str("Followup", registerRequest.Followup).
+					Msg("Machine is waiting for interactive login")
```

Comment:
```suggestion
				log.Debug().
					Caller().
					Str("machine", registerRequest.Hostinfo.Hostname).
					Str("node_key", registerRequest.NodeKey.ShortString()).
					Str("node_key_old", registerRequest.OldNodeKey.ShortString()).
					Str("follow_up", registerRequest.Followup).
					Msg("Machine is waiting for interactive login")
```

Keep the keys snakecased as the rest of our logs